libraries/ms-helper/src/thermistor.c

```diff
@@ -43,27 +42,30 @@ uint32_t thermistor_get_temp(ThermistorStorage *storage) {
       (storage->sibling_resistance / reading) * (vdda)-storage->sibling_resistance;
 
   // find the approximate target temperature
-  for (uint16_t i = 0; i < SIZEOF_ARRAY(resistance_lookup); i++) {
-    if (thermistor_resistance <= resistance_lookup[i] &&
-        thermistor_resistance >= resistance_lookup[i + 1]) {
+  for (uint16_t i = 0; i < SIZEOF_ARRAY(s_resistance_lookup) - 1; i++) {
```

Comment:
Don't duplicate code. You have a function that does the exact same thing.

Reply:
True